telematic_system/telematic_historical_data_processing/rosbag2_processing_service/rosbag2_processing_service/rosbag_processor.py

```diff
@@ -138,6 +139,8 @@ def create_record_from_msg(self, msg, measurement_name):
 
             elif isinstance(attr_value, list):  # Handle arrays
                 records.append(f'{attr_name}="{str(attr_value)}"')
+            elif isinstance(attr_value, bytes):
```

Comment:
isn't the bytes and list values conversion the same?

Reply:
Yes, this was avoid casting the byte string as a string. With this change, the value written to influx for byte strings is (b"...")